### PR TITLE
Bugfix: Avoid duplicated birthday events

### PR DESCRIPTION
**Pull request by annando**
Some users have reported duplicate birthday events. This fixes it.

But it seems that there is another issue with the birthdays that caused this problem. This is yet to be found. (But not in this issue)

### CODE REVIEW

File: include/event.php [hunk 1]
```diff
@@ -516,9 +535,9 @@ function event_by_id($owner_uid = 0, $event_params, $sql_extra = '') {
 		intval($event_params["event_id"])
 	);
 
-	if (dbm::is_result($r))
-		return $r;
-
+	if (dbm::is_result($r)) {
+		return event_remove_duplicates($r);
```
Comment (rabuzarus) [inline include/event.php]: Well one question: wouldn't it be better to, remove the duplicated entries from the db and avoid the creation of duplicates than to remove the duplicates every time from the array?

Comment (annando) [inline include/event.php]: Yeah, it would be better. This is a quick fix.

Comment (rabuzarus) [inline include/event.php]: OK. Can you please add a `/// @todo` as reminder

Comment (annando) [inline include/event.php]: Done. Maybe we can change this before the release. But I would like to concentrate now on the other issues.

**Pull request merged**